locust/webui/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "webui",
+  "version": "1.0.0",
+  "private": true,
+  "license": "MIT",
+  "scripts": {
+    "dev": "vite",
+    "dev:watch": "vite build --watch",
+    "build": "tsc && vite build",
+    "lint": "eslint './src/**/*.{ts,tsx}'",
+    "format": "prettier --write '**/**/*.{ts,tsx}'",
+    "type-check": "tsc"
+  },
+  "devDependencies": {
```

Comment:
How comfortable are you with the stability of these packages?

Leaving carets (^) in, in production is quite optimistic, and we risk builds and tests breaking overnight with no changes because some packages messed up a semver change.

Reply:
It might be better to uprev these on some duty cycle when performing maintenance upgrades.

Reply:
@mquinnfd We are commiting the `yarn.lock`, so we should not be worried about unexpected updates

Reply:
Yeah fair enough 👍